plugins/modules/cloud/scaleway/scaleway_container_registry.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Container registry management module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_container_registry
+short_description: Scaleway Container registry management module
+version_added: 5.8.0
+author: Guillaume MARTINEZ (@Lunik)
+description:
+  - This module manages container registries on Scaleway account.
+extends_documentation_fragment:
+  - community.general.scaleway
+  - community.general.scaleway_waitable_resource
+
+
+options:
+  state:
+    type: str
+    description:
+      - Indicate desired state of the container regitry.
+    default: present
+    choices:
+      - present
+      - absent
+
+  project_id:
+    type: str
+    description:
+      - Project identifier.
+    required: true
+
+  region:
```

Comment:
I see this comes from `SCALEWAY_REGIONS`. It might make sense to create another PR later and have a doc fragment generated automatically from that same list. Just a suggestion for the future ;-)

Reply:
Since some modules use the old API `scaleway_compute` and some the new API like this one (or `scaleway_database_backup`) It is not easy to put this value in a `doc_fragment`.

Reply:
Not familiar with the changes, taking your word for it. :-)